gfw_pixetl/utils/utils.py

```diff
@@ -28,6 +32,36 @@ def __init__(self, dst: Dict) -> None:
         self.metadata: Dict = {}
 
 
+def create_empty_file(work_dir, dst_profile: Dict[str, Any]):
+    # FIXME: This grabs the dtype and nodata value from dst_profile,
+    # but I think it should ACTUALLY copy those values from the profile
+    # of one of the input files in case they are different (for example
+    # if the source files are int32 and the desired output is uint16).
+    # It matters because there currently seems to be a problem mixing
+    # dtypes in one VRT.
+    local_file_path = os.path.join(work_dir, "input", "empty_file.tif")
+    profile = {
+        "driver": "GTiff",
+        "dtype": dst_profile["dtype"],
+        "nodata": dst_profile["nodata"],
+        "count": 1,
+        "width": 360,
+        "height": 180,
+        "crs": CRS.from_epsg(4326),  # FIXME: Always true?
+        "transform": Affine(1, 0, -180, 0, -1, 90),
```

Comment:
To be projection agnostic you could infer the input projection from other bands, set `width` and `height` to 1, the top left corner to the coordinate of the tile in the input projection and the pixel width and height to the tile size in input projection

Reply:
or simply copy the profile from a different layer which does intersect and use that

Reply:
Did the latter.